README.md

```diff
@@ -347,6 +347,13 @@ class PostSerializer < ActiveModel::Serializer
 end
 ```
 
+## Documentation
+
+AMS has two main sets of documentation: the
+[Docs](https://github.com/rails-api/active_model_serializers/tree/master/docs),
+which is a comprehensive guide and in
+[RDoc](http://www.rubydoc.info/github/rails-api/active_model_serializers) format.
```

Comment:
Thanks for this, :+1: 

should be above the `release  candidate plz read`. Also, I don't know if we can link to different versions of docs, or just note these are for master.